include/dynd/dispatcher.hpp

```diff
@@ -0,0 +1,250 @@
+//
+// Copyright (C) 2011-15 DyND Developers
+// BSD 2-Clause License, see LICENSE.txt
+//
+
+// #include <sparsehash/dense_hash_map>
+
+#include <dynd/types/type_id.hpp>
+
+namespace dynd {
+
+bool supercedes(const std::vector<type_id_t> &lhs, const std::vector<type_id_t> &rhs)
```

Comment:
There's some ambiguity as to whether this should be spelled supersede or supercede. Spelling it with an "s" seems to be slightly preferable (http://grammarist.com/spelling/supersede-or-supercede/).